<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>516</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>563</width>
    <height>268</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>979</width>
    <height>526</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>qESPTool GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comPorts">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshComPorts">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCheckBox" name="fwCheckBox">
        <property name="text">
         <string>Firmware</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="fsCheckBox">
        <property name="text">
         <string>SPIFFS</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="fwOpenDialog">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="fsFileLoc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="fsOpenDialog">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="fsAddrLbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Address 0x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="fwAddrLbl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Address 0x</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="fwFileLoc">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="fwAddr">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00000000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="fsAddr">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00300000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="3" rowspan="2">
       <widget class="QPushButton" name="uploadBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Reset Method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="resetMethod">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ck</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>wifio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>nodemcu</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dtrset</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="baudRate">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>256000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>512000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>921600</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Upload Speed:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QTextEdit" name="esptoolOut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>fwCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fwFileLoc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fwCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fwOpenDialog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fwCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fwAddr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fwCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fwAddrLbl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fsFileLoc</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fsOpenDialog</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fsAddr</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fsAddrLbl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
